python/semantic_kernel/connectors/ai/azure_ai_inference/azure_ai_inference_prompt_execution_settings.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from pydantic import Field
+
+from semantic_kernel.connectors.ai.prompt_execution_settings import PromptExecutionSettings
+from semantic_kernel.utils.experimental_decorator import experimental_class
+
+
+@experimental_class
+class AzureAIInferencePromptExecutionSettings(PromptExecutionSettings):
+    """Azure AI Inference Prompt Execution Settings."""
+
+    frequency_penalty: float | None = Field(None, ge=-2, le=2)
+    max_tokens: int | None = Field(None, gt=0)
+    presence_penalty: float | None = Field(None, ge=-2, le=2)
+    seed: int | None = None
+    stop: str | None = None
+    temperature: float | None = Field(None, ge=0.0, le=1.0)
+    top_p: float | None = Field(None, ge=0.0, le=1.0)
+    # `extra_parameters` is a dictionary to pass additional model-specific parameters to the model.
+    extra_parameters: dict[str, str] | None = None
+
+
+@experimental_class
+class AzureAIInferenceChatPromptExecutionSettings(AzureAIInferencePromptExecutionSettings):
+    """Azure AI Inference Chat Prompt Execution Settings."""
+
+
+@experimental_class
+class AzureAIInferenceEmbeddingPromptExecutionSettings(PromptExecutionSettings):
+    """Azure AI Inference Embedding Prompt Execution Settings."""
+
+    dimensions: int | None = Field(None, gt=0)
+    # Known values for `encoding_format` are
+    # "base64", "binary", "float", "int8", "ubinary", and "uint8".
+    encoding_format: str | None = None
```

Comment:
for known values, we should use a Literal, see OpenAIEmbeddingPromptExecutionSettings!